src/Inventory.zig

```diff
@@ -329,6 +329,21 @@ pub const Sync = struct { // MARK: Sync
 
 					inventory.inv.loadFromZon(inventoryZon);
 				},
+				.hand => {
+					const dest: []u8 = main.stackAllocator.alloc(u8, std.base64.url_safe.Encoder.calcSize(user.name.len));
+					defer main.stackAllocator.free(dest);
+					const hashedName = std.base64.url_safe.Encoder.encode(dest, user.name);
+
+					const path = std.fmt.allocPrint(main.stackAllocator.allocator, "saves/{s}/players/{s}.zig.zon", .{main.server.world.?.name, hashedName}) catch unreachable;
+					defer main.stackAllocator.free(path);
+		
+					const playerData = main.files.readToZon(main.stackAllocator, path) catch .null;
+					defer playerData.deinit(main.stackAllocator);
+
+					const inventoryZon = playerData.getChild("hand");
+
+					inventory.inv.loadFromZon(inventoryZon);
+				},
```

Comment:
You could merge this into the above switch case to avoid duplication:
```
.playerInventory, .hand => {
    ...
    const inventoryZon = playerData.getChild(@tagName(source));
    ...
},
```